### PR TITLE
Remove developer commands from server core (use Genisys-DevTools instead)

### PR DESCRIPTION
**Pull request by dktapps**
### Description
Remove developer commands (/makeplugin, /makeserver, /extractplugin etc.) from the server core as they are useless to the majority of end users and simply take up unnecessary space in the /help listings.
Instead, developers may use [Genisys-DevTools](https://github.com/iTXTech/Genisys-DevTools) for plugin/server development. I have also updated the CI configuration to make CI use Genisys-DevTools also.

### Reason to modify
- Less clutter in /help of commands which most people will find useless

### Tests & Reviews
I have tested the changes and everything works **as far as I can tell**. Please let me know if I missed anything or if anything is broken.

### CODE REVIEW

**Comment by Muqsit** [comment]
+1 Yeah, most of the consumers don't even know how to sendMessage(). (Most of them aren't even devs).

**Comment by dktapps** [comment]
@Muqsit exactly, these were just 5 extra unnecessary commands. Relocated them back to a plugin where they belong.

File: tests/ci.sh [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
+mkdir plugins
```
Comment (ishitatsuyuki) [inline tests/ci.sh]: You can remove if it works without this line.

File: src/pocketmine/resources/pocketmine.yml [hunk 2]
```diff
@@ -24,6 +24,8 @@ settings:
  #Set this approximately to your number of cores.
  #If set to auto, it'll try to detect the number of cores (or use 2)
  async-workers: 4
+ #Allow use of non-production builds. WARNING: Do not enable this unless you know what you are doing.
+ enable-testing: false
```
Comment (ishitatsuyuki) [inline src/pocketmine/resources/pocketmine.yml]: Wait, one change per commit.

**Comment by ishitatsuyuki** [comment]
Seems you have mixed something else into it. Please squash along.

**Comment by dktapps** [comment]
@ishitatsuyuki that was for the unsupported builds, I noticed it was missing. Shall I remove it?

**Comment by dktapps** [comment]
And the `mkdir` was required, it seems.

**Comment by dktapps** [comment]
Removed the pocketmine.yml change (will commit directly) and squashed commits. Anything else?

**Comment by ishitatsuyuki** [comment]
Looks good. Approved.

**Comment by gaoyichuan** [comment]
Approved and merge

**Pull request merged**